### DISCUSSION TITLE
Speakers for HyperSpy Workshop at the European Microscopy Congress 2024

### DISCUSSION THREAD

**Discussion by francisco-dlp** [hyperspy/hyperspy#3399] [General]
There will be a HyperSpy Workshop at the [EMC 2024](https://emc2024.eu/) on the 25th of August. The draft of them program is:

9:00-9:15 Intro talk
9:15-10:00 Intro tutorial
10:00-10:45 Decomposition and BSS
10:45-11:00 Coffee break
11:00-11:45 Curve fitting
11:45-12:30 EDX
12:30-13:45 Lunch break
13:45-14:30 EELS
14:30-15:15 Lumispy (Gunnar Kusch, University of Cambridge)
15:15-15:30 Coffee break
15:30-16:15 Lazy signals
16:15-17:00 Pyxem

@magnunor and I are organising it. If you are attending the EMC and you are interested in presenting one of the demos, let us know by discussing here.

**Comment by magnunor** [comment]
I can definitely do EELS, Lazy Signals and pyXem (at least parts of it).

If noone else can do it, I can also do EDX and the Intro tutorial.

@sivborg, @tinabe, @grigoryn94

**Comment by zezhong-zhang** [comment]
I will be at EMC, just wondering if it would be interested to mentioned the recently developed [Dirac GOS](https://github.com/hyperspy/exspy/pull/72) in EELS?

**Reply by ericpre** [reply]
Usually, existing notebooks (for example from https://github.com/hyperspy/exspy-demos) are being used/adapted. The GOS DFT cross section are mentioned in https://github.com/hyperspy/exspy-demos/blob/main/EELS/EELS_elemental_mapping.ipynb and it may be worth updating this notebook, to mention the GOS Dirac cross section. Before that, it would need a release of exspy.

**Reply by zezhong-zhang** [reply]
Thanks for directing to the notebook demo, I will make a PR to includes the recent changes.

**Reply by zezhong-zhang** [reply]
I did a simple PR with limited changes in https://github.com/hyperspy/exspy-demos/pull/8#issuecomment-2307404467.
